script.js

```diff
@@ -0,0 +1,9 @@
+// Display the prompt dialogue while the value assigned to `secret` is not equal to "sesame"
+let secret;
```

Comment:
I don't this this line is needed since it doesn't make sense to assign the prompt to this variable, you may delete this line

```suggestion

```